web-common/src/components/data-graphic/elements/SimpleDataGraphic.svelte

```diff
@@ -34,7 +35,7 @@ A simple composable container for SVG-based data graphics.
   export let shareYScale = true;
 
   export let mouseoverValue: DomainCoordinates | undefined = undefined;
-  export let hovered = undefined;
+  export let hovered = false;
```

Comment:
@djbarnwal -- I believe that hover state should always default to `false` unless explicitly set (here and in other files), rather than using undefined. Please LMK if that will cause any problems

Reply:
Sounds about right. Shouldn't cause any issue.